rpc/errdetails/error_details.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package infoblox.rpc;
+package atlas.rpc;
```

Comment:
this may effect recent changes for error handling. this need to be properly tested!